networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobsSidecarsByRangeMessageHandler.java

```diff
@@ -78,32 +77,39 @@ public BlobsSidecarsByRangeMessageHandler(
             "Total number of sidecars requested in accepted blobs sidecars by range requests from peers");
   }
 
+  /**
+   * See <a
+   * href="https://github.com/ethereum/consensus-specs/blob/dev/specs/eip4844/p2p-interface.md#blobssidecarsbyrange-v1">BlobsSidecarsByRange
+   * v1</a>
+   */
   @Override
   public Optional<RpcException> validateRequest(
       final String protocolId, final BlobsSidecarsByRangeRequestMessage request) {
-    final UInt64 maxSlotRequested = request.getMaxSlot();
-    final SpecMilestone specMilestone =
-        spec.getForkSchedule().getSpecMilestoneAtSlot(maxSlotRequested);
-    if (!specMilestone.isGreaterThanOrEqualTo(SpecMilestone.EIP4844)) {
+    final UInt64 requestEpoch = spec.computeEpochAtSlot(request.getStartSlot());
+    if (!verifyRequestEpochIsWithinMinEpochsForBlobsSidecars(requestEpoch)) {
```

Comment:
We could have blobs outside the `must` range and I'm not sure why we shouldn't handle such kind of requests. We are not forced to prune old blobs and some user may want to run archive blob nodes. We must have blobs in range but could do anything we want outside the range